Sources/Parley/Data/Models/ParleyLocalImage.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+public struct ParleyLocalImage: Codable, Identifiable, Equatable {
```

Comment:
Shall we rename to ParleyStoredImage?

Docs:
/// This can contain an image that we send with offline messaging, which has not reached the server yet. But can also contain an image that we retrieved from the server and store for recreating the chat when offline messaging is enabled.